.gitpod.yml

```diff
@@ -0,0 +1,9 @@
+tasks:
+  - init: make build
```

Comment:
```suggestion
  - init: make build-linux
```
We only really need to build linux binaries for something like this.

Reply:
OK